src/gui/properties/speedplotview.cpp

```diff
@@ -33,15 +33,61 @@
 #include "base/global.h"
 #include "base/utils/misc.h"
 
+namespace
+{
+    enum PeriodInSeconds
```

Comment:
Honestly, I don't understand why it was done in such an "archaic" way...
Not your issue.